enhancements/storage/csi-vsphere-operator.md

```diff
@@ -152,6 +152,62 @@ If we detect an existing driver present in the cluster then cluster will be mark
 
 We will ensure that these steps are documented in 4.10 docs.
 
+#### Configure CSI driver topology
+
+A customer may configure a topology for the CSI driver by using following fields in `ClusterCSIDriver` object:
```

Comment:
What part of this is talking about new fields and what part is context based on existing fields?

Reply:
updated it to reflect field being used to configure the topology. Also ended up renaming `topology_categories` to `tagCategories`. Shorter and sticks to more of k8s conventions (i guess).